### PR TITLE
[MIG][15.0] website: migration script

### PR DESCRIPTION
**Pull request by tranngocson1996**
Migration done

### CODE REVIEW

**Comment by tranngocson1996** [comment]
> /ocabot migration done

@legalsylvain /ocabot migration website

**Comment by legalsylvain** [comment]
Yes sorry ....

/ocabot migration website

**Comment by njeudy** [comment]
this https://github.com/OCA/OpenUpgrade/pull/3454 is waiting website to merge. Just for reference

File: openupgrade_scripts/scripts/website/15.0.1.0/upgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+new model theme.ir.asset
+new model website.configurator.feature
+new model website.searchable.mixin [abstract]
+# NOTHING TO DO
+
+---Fields in module 'website'---
+website      / ir.asset                 / key (char)                    : NEW
+website      / ir.asset                 / theme_template_id (many2one)  : NEW relation: theme.ir.asset
+website      / ir.asset                 / website_id (many2one)         : NEW relation: website
+# NOTHING TO DO
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/website/15.0.1.0/upgrade_analysis_work.txt:11]: Do you know if these fields are expected to be filled for existing records?

File: openupgrade_scripts/scripts/website/15.0.1.0/upgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+website      / theme.ir.asset           / active (boolean)              : NEW hasdefault
+website      / theme.ir.asset           / bundle (char)                 : NEW required
+website      / theme.ir.asset           / copy_ids (one2many)           : NEW relation: ir.asset
+website      / theme.ir.asset           / directive (selection)         : NEW selection_keys: ['after', 'append', 'before', 'include', 'prepend', 'remove', 'replace'], hasdefault
+website      / theme.ir.asset           / key (char)                    : NEW
+website      / theme.ir.asset           / name (char)                   : NEW required
+website      / theme.ir.asset           / path (char)                   : NEW required
+website      / theme.ir.asset           / sequence (integer)            : NEW required, req_default: function, hasdefault
+website      / theme.ir.asset           / target (char)                 : NEW
+# NOTHING TO DO: new model
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/website/15.0.1.0/upgrade_analysis_work.txt:25]: Shouldn't this be populated for existing installed themes?

File: openupgrade_scripts/scripts/website/15.0.1.0/upgrade_analysis_work.txt [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+
+website      / website                  / _order                        : _order is now 'sequence, id' ('id')
+website      / website                  / configurator_done (boolean)   : NEW
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/website/15.0.1.0/upgrade_analysis_work.txt]: Better to put this to `True`, isn't it?

File: openupgrade_scripts/scripts/website/15.0.1.0/upgrade_analysis_work.txt [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+website      / website                  / sequence (integer)            : NEW hasdefault
+# NOTHING TO DO
+
+website      / website                  / custom_code_footer (text)     : type is now 'html' ('text')
+website      / website                  / custom_code_head (text)       : type is now 'html' ('text')
+website      / website                  / robots_txt (text)             : type is now 'html' ('text')
+# DONE: pre-migation: convert custom_code_footer, custom_code_head, robots_txt field from text to html
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/website/15.0.1.0/upgrade_analysis_work.txt]: No transformation should be done here. They are still shown the same way as XML with widget=ace.

File: openupgrade_scripts/scripts/website/15.0.1.0/upgrade_analysis_work.txt [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+website      / website.configurator.feature / iap_page_code (char)          : NEW
+website      / website.configurator.feature / icon (char)                   : NEW
+website      / website.configurator.feature / menu_company (boolean)        : NEW
+website      / website.configurator.feature / menu_sequence (integer)       : NEW
+website      / website.configurator.feature / module_id (many2one)          : NEW relation: ir.module.module
+website      / website.configurator.feature / name (char)                   : NEW
+website      / website.configurator.feature / page_view_id (many2one)       : NEW relation: ir.ui.view
+website      / website.configurator.feature / sequence (integer)            : NEW
+website      / website.configurator.feature / website_config_preselection (char): NEW
+# NOTHING TO DO: new model
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/website/15.0.1.0/upgrade_analysis_work.txt]: Please include more explanation:

```suggestion
# NOTHING TO DO: new model for showing website features that is fed through <data> in the modules.
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]
I have pushed the refinements of my comments and a rebased branch, so ready to be merged if green.

**Pull request merged**